packages/shared-lib/src/core/model/StudioRouteSet.ts

```diff
@@ -18,7 +18,13 @@ export interface StudioRouteSet {
 	behavior: StudioRouteBehavior
 
 	routes: RouteMapping[]
+	/**
+	 * AB Pool members
+	 * Any players referenced here will only be active when this or another routeset is `active`
```

Comment:
Im a tad confused, what do you mean with "this _or another_ routeset is active"?